squire_core/src/state/mod.rs

```diff
@@ -125,8 +125,8 @@ impl AppStateBuilder<Database, ()> {
     pub fn build(self) -> AppState {
         let tourn_coll: Arc<str> = Arc::from(self.get_tournament_collection_name());
         let tourn_db = TournDb::new(self.db_conn.clone(), tourn_coll);
-        let tourns = ActorClient::builder(TournPersister::new(tourn_db.clone())).launch();
-        let gatherings = ActorBuilder::new(GatheringHall::new(tourns.clone())).launch();
+        let tourns = ActorBuilder::new(TournPersister::new(tourn_db.clone())).launch();
+        let gatherings = ActorBuilder::new(GatheringHall::<TournPersister>::new(tourns.clone())).launch();
         AppState {
```

Comment:
So just to be sure of my reasoning here: it looks like `sessions`, `accounts`, and `gatherings` are clients attached the app state, so those handlers should be `Permanent`. I think that `tourns` stays attached to `gatherings`, which means that that should also be `Permanent.`.

Reply:
Correct. All of those should be permanent. The only actors that should be transient are the `Gathering`s, which are internally managed by the `GatheringHall` actor.